Makefile

```diff
@@ -51,6 +51,8 @@ deploy: assets
 	node ./_deploy/compile-layouts
 	node ./_deploy/s3.js
 	$(MAKE) npm-publish
+	sleep 20
+	nht rebuild --all --serves user-page
```

Comment:
I'm in two minds as to whether this is a good thing or not (but happy to try/see how it goes)